osf/models/mixins.py

```diff
@@ -1593,7 +1593,10 @@ def _get_spam_content(self, saved_fields):
         spam_fields = self.get_spam_fields(saved_fields)
         content = []
         for field in spam_fields:
-            content.append((getattr(self, field, None) or '').encode('utf-8'))
+            exclude_null = {field + '__isnull': False}
+            values = list(self.__class__.objects.filter(id=self.id, **exclude_null).values_list(field, flat=True))
```

Comment:
This is an interesting solution - I agree with you that it will be nice to get rid of the Forward addon altogether at some point in the future, but I think defining this `addons_forward_node_settings__url` as a spam field on the node works well in the interim.  Can you add a detailed docstring to this method that explains the necessity for this unusual query?